controller/konnect/ops/mock/sdkfactory_mock.go

```diff
@@ -1,9 +1,11 @@
-package ops
+package mock
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
+
+	sdkops "github.com/kong/gateway-operator/controller/konnect/ops/sdk"
```

Comment:
Might as well as this to `golangci-lint` config to enforce the same import alias: https://github.com/Kong/gateway-operator/blob/a44e9a9f0b4b4e03baca0347fb7ec66e6036d073/.golangci.yaml#L43-L67

Reply:
https://github.com/Kong/gateway-operator/pull/764/commits/028ab7a5ab2e87a0d5dd1187cb2ba42b6f0b0854